### PR TITLE
Project status update

### PR DESCRIPTION
**Pull request by jamesmcroft**
## PR summary

Changes have been made to update the core project status documentation including the readme, contribution guidelines, issue and feature request templates, PR template, and included a standard .editorconfig file for code styling. 

- [ ] UWP
- [ ] Android
- [ ] iOS
- [x] CI/CD

## PR checklist

- [ ] Tests have been added/updated and pass
- [ ] Code styling has been run on all new source file changes
- [ ] Wiki documentation has been added
- [ ] Code sample has been added
- [x] Contains **NO** breaking changes

No breaking changes made.

## References

No additional references.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -1,32 +1,26 @@
...
-- [ ] New feature
-- [ ] Improvement to existing feature
-- [ ] Other (include details)
+## How will this benefit you and others?
+<!-- Please describe what benefit the introduction of the feature will bring to your project -->
 
-## What is the behaviour that will be introduced?
-> Please describe the change you'd like to see introduced
+## What is the expected API coverage?
+<!-- Please provide a list of the events and methods expected in the covered API you'd expect to see implemented -->
```
Comment (tom-made) [inline .github/ISSUE_TEMPLATE/feature_request.md]: This comment asks to list event and methods but the sections provided include properties. 

Is this intentional?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
 <img src="Assets/ProjectBanner.png" alt="XPlat Windows APIs" />
 
-# Contributing to XPlat Windows APIs
+# Contributing
 
-These guidelines are designed to help you as a contributor to bring your code and skills to the project to make it better.  
+These guidelines are designed to help you as a contributor to bring your code and skills to the XPlat Windows API project to make it better.
```
Comment (tom-made) [inline CONTRIBUTING.md]: Should this read XPlat Windows APIs project?

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (jamesmcroft) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Added an update to this to include properties. Well spotted

File: CONTRIBUTING.md (hunk 2 above)
Comment (jamesmcroft) [reply CONTRIBUTING.md]: Updated to read as XPlat Windows APIs.

**Pull request merged**